src/locale/ro/index.js

```diff
@@ -11,19 +11,19 @@
  * @language Romanian
  * @iso-639-2 ron
  * @author Sergiu Munteanu [@jsergiu]{@link https://github.com/jsergiu}
+ * @author Adrian Ocneanu [@aocneanu]{@link https://github.com/aocneanu}
+ * @author Mihai Ocneanu [@aocneanu]{@link https://github.com/gandesc}
```

Comment:
Please fix the link to the author:

```suggestion
 * @author Mihai Ocneanu [@gandesc]{@link https://github.com/gandesc}
```